src/js/dialogs/sandbox.js

```diff
@@ -20,7 +20,7 @@ exports.dialog = {
         'Try out ',
         '[this special link](https://pcottle.github.io/learnGitBranching/?NODEMO)',
         '',
-        'PPS: GitHub (and the industry at large) is moving to naming the default branch as `main` instead of `master` ',
+        'PPS: GitHub (and the industry at large) is moving to naming the default branch as `main` instead of `main` ',
```

Comment:
@pcottle @hong4rc this one also should not have been changed! 😄

Reply:
Ah yeah good catch -- will update

Reply:
https://github.com/pcottle/learnGitBranching/commit/ab3a2349edb62e8d396b64930aed59972e2486a2

Reply:
<img width="1049" alt="Screen Shot 2021-04-19 at 1 14 35 PM" src="https://user-images.githubusercontent.com/1135007/115290894-52c39f00-a111-11eb-939e-b96693d99e9c.png">